frontend/src/metabase/css/core/colors.module.css

```diff
@@ -130,6 +130,13 @@
     white 90%
   );
 
+  /* Saturated colors for the SQL editor. Shouldn't be used elsewhere since they're not white-labelable. */
+  --mb-color-saturated-blue: #2d86d4;
+  --mb-color-saturated-green: #70a63a;
+  --mb-color-saturated-purple: #885ab1;
+  --mb-color-saturated-red: #ed6e6e;
+  --mb-color-saturated-yellow: #f9cf48;
```

Comment:
These colors only existed in `frontend/src/metabase/lib/colors/palette.ts` and not here.

Reply:
the `mb-saturated` colors get used elsewhere too (eg. in `frontend/src/metabase/admin/tasks/components/Logs/Logs.styled.tsx`).
Moving them to the ace editor component might cause them to go out of sync.

I agree on using the `--mb-base-color` colors instead however! I'll set up a PR with a new color scheme proposal after this one, but I don't think it should block this PR.